lib/puppet/parser/functions/platform_fretta.rb

```diff
@@ -25,11 +25,13 @@ module Parser
     #
     module Functions
       newfunction(:platform_fretta, type: :rvalue) do |_args|
-        data = lookupvar('os')
+        data = lookupvar('cisco')
         return '' if data.nil?
 
-        pat = '7.0\(3\)F'
-        data['release']['full'].match(pat) ? true : false
+        data['inventory'].each do |_x, slot|
```

Comment:
Did you verify the demo manifest with this change?

Reply:
Yes